homeworks/Dmitriy.Tyulpa_innocentDimon/homework_7/task.test.js

```diff
@@ -0,0 +1,26 @@
+const { fibonachi } = require('./task.js');
+
+describe('Task test', () => {
+    test('Type array', () => {
+        expect(Array.isArray(fibonachi())).toBe(true);
+    });
+
+    test('First element', () => {
+        expect(fibonachi()[0]).toBe(0);
+    });
+
+    test('Five element', () => {
+        expect(fibonachi()[4]).toBe(3);
+    });
+
+    test('Ten element', () => {
+        expect(fibonachi()[9]).toBe(34);
+    });
+
+    test('Sum test', () => {
+        const num = fibonachi();
+        for (let i = 2; i < num.length; i++) {
+            expect(num[i]).toBe(num[i - 1] + num[i - 2]);
+        }
+    });
+});
```

Comment:
it would be nice to have test description more informative, e.g. 'first element of array should be 0'

when you run a list of 1000 tests and some test fails it's good to know what exactly doesn't work correctly